### PR TITLE
Update ethToLend to wethToLend

### PR DESCRIPTION
**Pull request by aleone**
# Task: Rename ethToLend/ ethFromEuler... to WethToLend/WethFromEuler...

## Description

Rename ethToLend/ ethFromEuler... to WethToLend/WethFromEuler...

Fixes # Fixes ENG-832

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

### CODE REVIEW

File: packages/bull-vault/src/LeverageBull.sol [hunk 1]
```diff
@@ -125,16 +125,16 @@ contract LeverageBull is Ownable {
...
-        (uint256 ethToLend, uint256 usdcToBorrow) = _calcLeverageEthUsdc(
+        (uint256 wethToLend, uint256 usdcToBorrow) = _calcLeverageEthUsdc(
             _crabAmount, _bullShare, _ethInCrab, _squeethInCrab, _crabTotalSupply
         );
 
-        require(ethToLend == _ethAmount, "LB0");
+        require(wethToLend == _ethAmount, "LB0");
 
-        _depositEthInEuler(ethToLend, true);
+        _depositEthInEuler(wethToLend, true);
```
Comment (haythem96) [inline packages/bull-vault/src/LeverageBull.sol:134]: `_depositEthInEuler` => `_depositWethInEuler`

File: packages/bull-vault/src/LeverageBull.sol [hunk 2]
```diff
@@ -125,16 +125,16 @@ contract LeverageBull is Ownable {
         uint256 _squeethInCrab,
         uint256 _crabTotalSupply
     ) internal returns (uint256, uint256, uint256) {
-        (uint256 ethToLend, uint256 usdcToBorrow) = _calcLeverageEthUsdc(
+        (uint256 wethToLend, uint256 usdcToBorrow) = _calcLeverageEthUsdc(
```
Comment (haythem96) [inline packages/bull-vault/src/LeverageBull.sol:128]: Maybe `_calcLeverageEthUsdc` to `_calcLeverageWethUsdc`

Comment (aleone) [reply packages/bull-vault/src/LeverageBull.sol:128]: happy to change this one as well

File: packages/bull-vault/src/LeverageBull.sol (hunk 1 above)
Comment (aleone) [reply packages/bull-vault/src/LeverageBull.sol:134]: happy to change this one as well

**Review by haythem96 — APPROVED** [review]
lgtm

**Pull request merged**